Makefile.dryice.js

```diff
@@ -627,6 +649,19 @@ function extractCss(callback) {
     }
 }
 
+function extractNls() {
+    var allMessages = {};
+    searchFiles(__dirname + "/src", function(path) {
+        var text = fs.readFileSync(path, "utf8");
+        var matches = text.match(/nls\s*\(\s*("([^"\\]|\\.)+"|'([^'\\]|\\.)+')/g);
+        matches && matches.forEach(function(m) {
+            var eng = m.replace(/^nls\s*\(\s*["']|["']$/g, "");
+            allMessages[eng] = "";
+        });
+    });
+    fs.writeFileSync(__dirname + "/messages.json", JSON.stringify(allMessages, null, 4), "utf8");
```

Comment:
what if this throws error?

Reply:
it will be printed in the terminal, and the developer running it will decide what to do.